content/sensu-go/5.2/reference/assets.md

```diff
@@ -41,18 +59,120 @@ If the requested asset is not in the local cache, it is downloaded from the asse
+For more information about creating Sensu Plugins, see the [Sensu Plugin specification][29].
+
+Bonsai requires a [`bonsai.yml` configuration file](#bonsai-yml-specification) in the root directory of your repository that includes the project description, platforms, asset filenames, and SHA-512 checksums.
+For a Bonsai-compatible asset template using Go and [GoReleaser](https://goreleaser.com/), see the [Sensu Go plugin skeleton][28].
+
+To share your asset on Bonsai, [log in to Bonsai](https://bonsai.sensu.io/sign-in) with your GitHub account and authorize Sensu.
+Once logged in, you can [register your asset on Bonsai](https://bonsai.sensu.io/new) by adding the GitHub repository, description, and tags.
+Make sure to provide a helpful README for your asset with configuration examples.
+
+### `bonsai.yml` example
+
+{{< highlight yml >}}
+---
+description: "#{repo}"
+builds:
+- platform: "linux"
+  arch: "amd64"
+  asset_filename: "#{repo}_#{version}_linux_amd64.tar.gz"
+  sha_filename: "#{repo}_#{version}_sha512-checksums.txt"
+  filter:
+  -  "entity.system.os == 'linux'"
+  -  "entity.system.arch == 'amd64'"
+
+- platform: "Windows"
+  arch: "amd64"
+  asset_filename: "#{repo}_#{version}_windows_amd64.tar.gz"
+  sha_filename: "#{repo}_#{version}_sha512-checksums.txt"
+  filter:
+  -  "entity.system.os == 'windows'"
+  -  "entity.system.arch == 'amd64'"
+{{< /highlight >}}
+
+### `bonsai.yml` specification
+
+ description | 
+-------------|------
+description  | The project description
+required     | true
+type         | String
+example      | {{< highlight yml >}}description: "#{repo}"{{< /highlight >}}
+
+ builds      | 
+-------------|------
+description  | An array of asset details per platform
+required     | true
+type         | Array
+example      | {{< highlight yml >}}
+builds:
+- platform: "linux"
+  arch: "amd64"
+  asset_filename: "#{repo}_#{version}_linux_amd64.tar.gz"
+  sha_filename: "#{repo}_#{version}_sha512-checksums.txt"
+  filter:
+  -  "entity.system.os == 'linux'"
+  -  "entity.system.arch == 'amd64'"
+{{< /highlight >}}
+
+### Builds specification
+
+ platform    | 
+-------------|------
+description  | The platform supported by the asset
+required     | true
+type         | String
+example      | {{< highlight yml >}}- platform: "linux"{{< /highlight >}}
+
+ arch        | 
+-------------|------
+description  | The architecture supported by the asset
+required     | true
+type         | String
+example      | {{< highlight yml >}}  arch: "amd64"{{< /highlight >}}
+
+asset_filename | 
+-------------|------
+description  | The filename of the archive containing the asset
+required     | true
+type         | String
+example      | {{< highlight yml >}}asset_filename: "#{repo}_#{version}_linux_amd64.tar.gz"{{< /highlight >}}
+
+sha_filename | 
+-------------|------
+description  | The SHA-512 checksum for the archive
```

Comment:
```suggestion
description  | The SHA-512 checksum for the asset archive
```

Not sure if that is the proper language. I understand an asset to be the entire package, not just the executable(s) inside. Suggestion made as all other attribute descriptions under the `spec` scope contain it.